src/net/sourceforge/kolmafia/preferences/Preferences.java

```diff
@@ -762,6 +762,10 @@ private static void encodeCharacter(char ch) {
                         : "\\u" + Integer.toHexString(ch);
   }
 
+  public static boolean propertyExists(final String name) {
+    return propertyExists(name, true) || propertyExists(name, false);
+  }
```

Comment:
This looks weird.  I might refactor so there were userPropertyExists(name) and globalPropertyExists(name) for readability but this is not a blocker nor even a firm well thought out preference.